js-plot/index.html

```diff
@@ -0,0 +1,36 @@
+<html>
+  <head>
+    <script type="text/javascript" src="newtonraphson.js"></script>
+    <script src="https://cdn.jsdelivr.net/npm/vega@5.13.0"></script>
+    <script src="https://cdn.jsdelivr.net/npm/vega-lite@4.13.0"></script>
+    <script src="https://cdn.jsdelivr.net/npm/vega-embed@6.8.0"></script>
+  </head>
+  <body>
+    <div id="plot"></div>
+    <script>
+      createModule().then(async (rootfinding) => {
+        const tolerance = 0.001;
+        const initial_guess = -4;
+        const newtonraphson = new rootfinding.NewtonRaphson(tolerance);
+        newtonraphson.solve(initial_guess);
+        const iterations = new Array(
```

Comment:
We need to explain what is happening between line 16 and 31. It won't be obvious for the readers that are not familiar with JavaScript.

Reply:
Indeed, added todo in 4162e1e7d370490dcb3f5325e26d1a326f9eb1e2